distro/configs/bahmniemr/eip/config/application.properties

```diff
@@ -0,0 +1,189 @@
+camel.springboot.xml-routes=classpath*:camel/*.xml,file:routes/*.xml,file:routes/*/*.xml,file:routes/*/*/*.xml
+
+openmrs.db.host=${OPENMRS_DB_HOST}
+
+# The OpenMRS MySQL database port
+openmrs.db.port=${OPENMRS_DB_PORT}
+
+# The OpenMRS MySQL database name
+openmrs.db.name=${OPENMRS_DB_NAME}
+# *********************** General EIP sender properties ****************************************************************
+#
+
+# Maximum size of each batch of source records to be read by debezium from the binlog files, maps to the
+## debezium MySQL connector property named max.batch.size
+#https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-property-max-batch-size
+debezium.reader.maxBatchSize=2048
+
+# The client Id of the account. Defaults to empty.
+oauth.access.token.uri=${OAUTH_ACCESS_TOKEN_URL:}
+
+# The client Id of the account to use to authenticate. Defaults to empty.
+oauth.client.id=${OAUTH_CLIENT_ID:}
+
+# The client secret of the account to use to authenticate. Defaults to empty.
+oauth.client.secret=${OAUTH_CLIENT_SECRET:}
+
+# Authentication scope, can be multiple values separated by commas. Defaults to empty.
+oauth.client.scope=${OAUTH_CLIENT_SCOPE:}
+
+#Authentication scope, can be multiple values separated by commas
+oauth.client.scope=email
+
```

Comment:
Have you tried running this with SSO enabled? I think you need at least `openid` scope

Reply:
This is simply a place holder, I don't think SSO is even necessary for this service as it only deals with the database directly, however the props need to be provided otherwise the app will fail.